sormas-backend/pom.xml

```diff
@@ -275,6 +275,16 @@
 			<artifactId>reflections</artifactId>
 		</dependency>
 
+		<dependency>
```

Comment:
The new dependencies for `org.apache.tika` are to be listed in `sormas-base/pom.xml` (alphabetically sorting in section "Compile dependencies")

Reply:
Done